src/utils/datatool/Datatool.cpp

```diff
@@ -95,13 +96,13 @@ int createGibsonSemanticMesh(const std::string& objFile,
   f << "end_header" << std::endl;
 
   // We need to rotate to match .glb where -Z is gravity
-  const auto transform =
-      esp::quatf::FromTwoVectors(esp::vec3f::UnitY(), esp::vec3f::UnitZ());
+  const Mn::Quaternion transform = esp::core::quatRotFromTwoVectors(
+      Mn::Vector3::yAxis(), Mn::Vector3::zAxis());
```

Comment:
what :eyes: 

```cpp
const Mn::Quaternion transform = Mn::Quaternion::rotation(90.0_degf, Mn::Vector3::xAxis());
```

Well, or just flip the components or something, this feels like a nuclear solution to the problem :D